api_v3/web/getFeed.php

```diff
@@ -73,6 +73,7 @@ function getRequestParameter($paramName)
 $entryId = getRequestParameter('entryId');
 $limit = getRequestParameter('limit');
 $ks = getRequestParameter('ks');
+$playerUrlType = getRequestParameter('playerUrlType');
```

Comment:
Why don't we add this parameter to the feed instead of accepting it in the request?
This way we can define that the now feeds from now on will return the new player URL.